dist/index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>TV-Series Showcase</title>
+  <script src="https://kit.fontawesome.com/ed14e3c3ce.js" crossorigin="anonymous"></script>
+<script defer src="runtime.js"></script><script defer src="main.js"></script></head>
+<body>
+  <header>
+    <nav>
+      <ul>
+        <li id="img-logo"><a href=""></a></li>
+        <li><a href="#">Awesome TV-Series</a></li>
+      </ul>
+    </nav>
+  </header>
+  <section class="main-sec" id="main-sec-id">
+     <!-- <div class="card" id="card-id#">
+      <img id="card-img#" class="img-class">
+      <div class="card-subtitle">
+        <h4 class="subtitle"></h4>
+        <i class="like-icon"></i>
+      </div>
+      <div class="likes">
+        <h5>Amount of likes</h5>
+      </div>
+      <button id="btn-id#" class="comment-btn">Comment</button>
+    </div>  -->
```

Comment:
>- You have a couple of commented code which you do not need, because they are redundant it is best practice to keep your codebase clean by removing the commented unused code.